src/parser/jobs/sch/modules/Overheal.js

```diff
@@ -6,6 +6,7 @@ import React from 'react'
 
 export default class Overheal extends CoreOverheal {
 	static handle = 'overheal'
+	static debug = true
```

Comment:
We don't allow committing debug = true
```suggestion
	static debug = false
```

Reply:
I know; I missed it. One minute and I'll fix.